include/swift/AST/Stmt.h

```diff
@@ -1336,13 +1336,13 @@ class BreakStmt : public Stmt {
   SourceLoc Loc;
   Identifier TargetName; // Named target statement, if specified in the source.
   SourceLoc TargetLoc;
-  LabeledStmt *Target = nullptr;  // Target stmt, wired up by Sema.
+  DeclContext *DC;
```

Comment:
Could you add a forward declaration for `DeclContext` in this `.h` file?
As far as I can see. No header in this file has `#include "swift/AST/DeclContext.h"`

Reply:
Good catch!